staging/src/k8s.io/client-go/CHANGELOG.md

```diff
@@ -0,0 +1,307 @@
+TODO: This document was manually maintained so might be incomplete. The
```

Comment:
this file can only move if we agree to integrate the client-go release process into the kube release process. This definitely needs further discussion. If we want to go this route, we need a published-repos roles in the release team with the authority to do whatever is needed during a release.

Reply:
Are there use cases for this that would require direct write access to kubernetes/kubernetes?
 If so, I would really appreciate it if you could articulate them here: https://github.com/kubernetes/kubernetes/issues/57667

Reply:
It's about agility during tagging of client-go, not so much about write access. Every change in k/k to show up in client-go needs a publishing-bot run (every few hours right now).

Reply:
And the client-go tags happen asynchronously to kube right now. Which means that we would want to update k/k branches after the official tags. This involves the release (branch) manager to agree on those.